content/doc/tutorials/using-jenkinsfile-runner-github-action-to-build-jenkins-pipeline.adoc

```diff
@@ -0,0 +1,308 @@
+[source,yaml]
+----
+name: Java CI
+on: [push]
+----
+* Use a Ubuntu runner for the job.
+[source,yaml]
+----
+jobs:
+  job-name:
+    runs-on: ubuntu-latest
+----
+* (Optional) If you want to use `jfr-container-action` later, you need to declare using the `ghcr.io/jenkinsci/jenkinsfile-runner:master` or any image extending it. 
+If you choose to use `jfr-static-image-action`, you can skip this step.
+[source,yaml]
+----
+jobs:
+  job-name:
+    runs-on: ubuntu-latest
+    container:
+      image: ghcr.io/jenkinsci/jenkinsfile-runner:master
+----
+* Call the `actions/checkout@v2` to pull your codes into the runner. 
+“Call” means “uses” in the workflow definition specifically. 
+You can check the details about the “uses” keyword.
+[source,yaml]
+----
+- uses: actions/checkout@v2
+----
+* Call the Jenkinsfile-runner actions.
+** If you use `jfr-container-action`, you need to call `jenkinsci/jfr-container-action@master` and give necessary inputs.
+[source,yaml]
+----
+uses:
+  jenkinsci/jfr-container-action@master
+with:
+  command: run
+  jenkinsfile: Jenkinsfile
+  pluginstxt: plugins.txt
+----
+** If you use `jfr-static-image-action`, you need to call `jenkinsci/jfr-static-image-action@master` and give necessary inputs.
+[source,yaml]
+----
+uses:
+  jenkinsci/jfr-static-image-action@master
+with:
+  command: run
+  jenkinsfile: Jenkinsfile
+  pluginstxt: plugins.txt
+----
+
+You can check the full example workflow definition for link:http://jfr-static-image-action[jfr-container-action] and link:http://jfr-static-image-action[jfr-static-image-action]. 
+There is another powerful GitHub Action list called Jenkinsfile Runner Runtime Actions. 
+For these GitHub Actions, you can run them in any runners, which are Windows, Linux and macOS. 
+You can check their step by step usage link:http://jfr-static-image-action[here].
+
+=== Access the workflow logs
+
+Once you create your workflow definition, you can commit all the local changes to the remote repository. 
+When your workflow is triggered, the workflow is executed. 
+After this workflow has started, you can see the visualization graph of the run's progress and view each step's activity on GitHub. 
+If you want to learn more details about viewing your workflow details, you can check link:https://docs.github.com/en/actions/quickstart#viewing-your-workflow-results[the official guide].
+[.boxshadow]
+image:tutorials/jenkinsfile-runner-github-actions-01-access-workflow-logs.jpeg[alt="Access the workflow logs",width=100%]
+
+=== Add JCasC (Optional)
+
+Usually we need to access the web UI to set up Jenkins. 
+However, we’re unable to access the web UI under the circumstances of running Jenkins pipeline in the GitHub Actions because the Jenkins instance is ephemeral. 
+JCasC (link:https://github.com/jenkinsci/configuration-as-code-plugin[Jenkins Configuration as Code]) plugin can configure this ephemeral Jenkins instance by providing the human-readable declarative configuration files. 
```

Comment:
```suggestion
Typically, we need to access the web UI to set up Jenkins. 
However, we’re unable to access the web UI under the circumstances of running Jenkins pipeline in the GitHub Actions, since the Jenkins instance is ephemeral. 
The JCasC (link:https://github.com/jenkinsci/configuration-as-code-plugin[Jenkins Configuration as Code]) plugin can configure this ephemeral Jenkins instance, by providing the human-readable declarative configuration files. 
```
syntax/formatting suggestions